### PR TITLE
Primitive multi card search

### PR DESCRIPTION
**Pull request by janbj**

### CODE REVIEW

File: base.css [hunk 1]
```diff
@@ -296,6 +296,11 @@ div.result-message {
   padding: 0;
   margin: 1em 0;
 }
+.clear-search-btn {
+  width: 10%;
+  margin: 0;
+  padding: 0;
+}
```
Comment (lionel-panhaleux) [inline base.css:303]: You could style the selected card names and clear button inline (or not)

File: deck-search/twda.js [hunk 2]
```diff
@@ -1,3 +1,44 @@
...
+        }
+        for (let card of this.cardList.values()) {
+            let cardNode = document.createElement('li')
+            cardNode.textContent = `${card}`
+            cardNode.addEventListener("click", function (e) {
+                removeMultiCardSearchCard(card)
+            })
+            cardListNode.append(cardNode)
+        }
+    }
```
Comment (lionel-panhaleux) [inline deck-search/twda.js:33]: Nothing that's really mandated here, but maybe including the search params in the URL (and window history), could be a cool feature. Maybe that can come next though 😋

Comment (janbj) [reply deck-search/twda.js:34]: We can register a new issue for this.

File: base.css (hunk 1 above)
Comment (janbj) [reply base.css:303]: Added better styling.

**Pull request merged**